<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegulatoryEfficacy</class>
 <widget class="QMainWindow" name="RegulatoryEfficacy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>910</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>板形调控功效挖掘</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qtResource/resource.qrc">
    <normaloff>:/icons/icons/反馈控制_black.svg</normaloff>:/icons/icons/反馈控制_black.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,7">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>7</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>模型计算模块</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,6">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QGroupBox" name="groupBox_8">
                 <property name="title">
                  <string>宽度选择</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QComboBox" name="comboBox">
                    <item>
                     <property name="text">
                      <string>800-900</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>900-1000</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>1000-1100</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>1100-1200</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>1200-1300</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>1300-1400</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>1400-1500</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>1500-1600</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>1600-1700</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>1700-1800</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>1800-1900</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_7">
                 <property name="title">
                  <string>轧辊粗糙度</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <widget class="QRadioButton" name="radioButton_3">
                    <property name="text">
                     <string>光辊</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_4">
                    <property name="text">
                     <string>毛辊</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QPushButton" name="pushButton_jisuan">
                   <property name="text">
                    <string>模型计算</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_qingkong">
                   <property name="text">
                    <string>清空</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QTextEdit" name="textEdit"/>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_10">
                 <item>
                  <widget class="QPushButton" name="pushButton_jisuanjieguo">
                   <property name="text">
                    <string>显示计算结果</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_xianshijieguo">
                   <property name="text">
                    <string>显示图像结果</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="focusPolicy">
             <enum>Qt::TabFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="title">
             <string>系数显示模块</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTableView" name="tableView"/>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QPushButton" name="pushButton_qingkongxianshijieguo">
                 <property name="text">
                  <string>清空</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_baocun">
                 <property name="text">
                  <string>保存</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>图形展示模块</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>7</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_6" stretch="6,1">
            <item>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <widget class="QGraphicsView" name="graphicsView_tuxing"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QPushButton" name="pushButton_qingkonghuabu">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>清空画布</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../qtResource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
